### DISCUSSION TITLE
New size-* utilities not working

### DISCUSSION THREAD

**Discussion by PunkFleet** [tailwindlabs/tailwindcss#12691] [Help]
I using Tailwindcss v3.4.0, and trying new features follow this: https://tailwindcss.com/blog/tailwindcss-v3-4#new-size-utilities
But it not working
```
 <img class="size-14 border-2 border-indigo-500" src="{% static 'admin/logo_300px.jpg' %}"  alt="logo"/>
```
and it is working:
```
  <img class="h-14 w-14 border-2 border-indigo-500" src="{% static 'admin/logo_300px.jpg' %}"  alt="logo"/>
```

I don't know why it is happend, what should i do?

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- You have updated Tailwind to `v3.4.0` or newer.
